functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/pages/TopMenuPage.groovy

```diff
@@ -17,16 +14,35 @@ class TopMenuPage extends BasePage {
 
     By settingsButtonBy = By.id("appAdmin")
     By systemConfigurationMenuBy = By.linkText("System Configuration")
+    By userManagerMenuBy = By.linkText("User Manager")
```

Comment:
if this is pro only, you could create a "Page" in PRO that extends from "TopMenuPage" to add this locator

Reply:
refactored!